### ISSUE TITLE
[Regression] Cannot start some programs since 0.18.0 - bad `main` handling.

### ISSUE THREAD

**Issue by MichalCz** [scramjetorg/transform-hub#314]
**Describe the bug**

Node.js sequences that point to main file without extension (`"main": "./index"`) won't start after sending.

**Steps to Reproduce**

Terminal 1:

```sh
npm i -g @scramjet/cli @scramjet/sth
SCRAMJET_TEST_LOG=1 DEVELOPMENT=1 sth --no-docker --identify-existing
```

Terminal 2:

```sh
git clone https://github.com/scramjetorg/scramjet-cloud-docs.git
cd scramjet-cloud-docs/samples/
si pack hello-snowman
si send -
si start -
```

Effect: SI process hangs, STH logs show error.

**Expected behavior**

Runner should import the file regardless of having or not having the extension.

**Version (please complete the following information):**
 - STH version: 0.18.0
 - node version:
 - os:

**Temporary workaround**

* Downgrading to 0.17.0 fixes the issue
* Adding `.js` in `package.json`/`main` fixes the issue.

**Comment by MichalCz** [comment]
Fixed as of 0.18.3

**Comment by MichalCz** [comment]
If you are affected, please update to the newest version using:

```
npm i -g @scramjet/sth # on the server
npm i -g @scramjet/cli # on the clients
```
